### ISSUE TITLE
Use release mozdownload instead of Hutu's repo

### ISSUE THREAD

**Issue by mwobensmith** [mozilla/iris2#68]
The pipfile currently points to Hutu's private repo for `mozdownload`:
```
mozdownload = {git = 'https://github.com/AndreiH/mozdownload.git',ref = 'mozdownload_python3'}
```
This needs to change before release.

**Comment by AndreiH** [comment]
This should be updated only when https://github.com/mozilla/mozdownload/pull/536 is merged

**Comment by AndreiH** [comment]
https://github.com/mozilla/mozdownload/pull/536 was merged and a new version of Mozdownload was released, version 1.26 https://github.com/mozilla/mozdownload/releases

**Comment by mwobensmith** [comment]
Fixed
